pyproject.toml

```diff
@@ -69,14 +69,15 @@ pgvector = {optional = true, version = "^0.1.0"}
 psycopg-binary = {optional = true, version = "^3.1.12"}
 optimum = {extras = ["onnxruntime"], optional = true, version = "^1.13.2"}
 sentencepiece = {optional = true, version = "^0.1.99"}
-transformers = {extras = ["torch"], optional = true, version = "^4.34.0"}
+transformers = {extras = ["torch"], optional = true, version = "^4.33.1"}
 guidance = {optional = true, version = "^0.0.64"}
 lm-format-enforcer = {optional = true, version = "^0.4.3"}
 jsonpath-ng = {optional = true, version = "^1.6.0"}
 rank-bm25 = {optional = true, version = "^0.2.2"}
 scikit-learn = {optional = true, version = "*"}
 spacy = {optional = true, version = "^3.7.1"}
 aiohttp = "^3.8.6"
+tensorrt_llm = {optional = true, python = "3.10", source = "nvidia-pypi", version = "^0.7.1"}
```

Comment:
do we want to move it to `[tool.poetry.group.dev.dependencies]` cc @logan-markewich

Reply:
No, dev-dependencies are for things needed during CI I think